Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -468,16 +471,15 @@ def channel_state(self):
                 return "Updating"
             if self.subscribed:
                 return "Downloading"
-            else:
-                return "Preview"
+            return "Preview"
 
         @db_session
         def to_simple_dict(self):
             """
             Return a basic dictionary with information about the channel.
             """
             epoch = datetime.utcfromtimestamp(0)
-
+            # TODO: optimize this?
```

Comment:
While I think adding `TODO` notes helps us to quickly identify possible performance bottlenecks, please add a bit more information to these notes (e.g., on how you think it could be optimized).

Reply:
Ok